configuration/analyzer_config.json

```diff
@@ -786,6 +786,26 @@
     "secrets": {},
     "params": {}
   },
+  "FileScan_Search": {
+    "type": "observable",
+    "python_module": "filescan_search.FileScanSearch",
+    "description": "Finds reports and uploaded files by various tokens, like hash, filename, verdict, IOCs etc via [FileScan.io API](https://www.filescan.io/api/docs).",
+    "disabled": false,
+    "external_service": true,
+    "leaks_info": false,
+    "observable_supported": [
+      "url",
+      "domain",
+      "ip",
+      "generic"
+    ],
+    "config": {
+      "soft_time_limit": 180,
+      "queue": "long"
+    },
+    "secrets": {},
+    "params": {}
```

Comment:
Add support for these parameters:

- `age`. type: `int`, default: `-1`.
- `method`. type: `str`, default: `and`.
- `page`. type: `int`, default: `1`.
- `page_size`. type: `int`, default: `50`.